cuegui/cuegui/config/cue_resources.yaml

```diff
@@ -26,3 +26,7 @@ max_proc_hour_cutoff: 30
 # cores is higher that this threshold. To disable this warning, set the 
 # threshold to -1
 redirect_wasted_cores_threshold: 100
+
+
+"frame.finished_jobs_readonly": False
+"layer.finished_jobs_readonly": False
```

Comment:
Please add a comment explaining what this config is for.

Reply:
Added comments in the most recent update